### ISSUE TITLE
Icons of buttons are not clearly visible in multiple places only in High Contrast 'Aquatic' mode.

### ISSUE THREAD

**Issue by msftedad** [tinymce/tinymce#8507]
**What is the current behavior? Describe the bug**
Icons are not clearly visible in the multiple places in High Contrast 'Aquatic' mode:
Users with vision impairment who rely on High Contrast Aquatic mode will find it difficult to see and access the buttons if icon of the button is not clearly visible.

**Pre-requisites:** Turn on High Contrast (Aquatic/Desert) using Settings->Accessibility->Contrast Themes->Aquatic/Desert.

**Please provide the steps to reproduce and if possible a minimal demo of the problem via [fiddle.tiny.cloud](https://fiddle.tiny.cloud/) or similar.**

Step 1: Please open https://codepen.io/keyapanja/pen/JjBMdBg in edge browser.
Step 2: The default focus would be on HTML body
Step 3: Press ESC, the focus would be on 'Open HTML settings'
Step 4:: Navigate to Codepen preview Area.
Step 5: Navigate to Formatting toolbar and observe the following issues.

**Issue1:** After invoking any of the menu options like 'File','Edit','Insert'...,->expand arrows(>) are not clearly visible. (Note: This issue is repro for all menu options having expand arrow(>)).

**Issue 2:** Navigate to Formatting toolbar>insert->link->dialog box->open link in.. combo box->dropdown arrow(v) not clearly visible. (Note : This issue is repro in all such dialog boxes in Formatting toolbar).

**Issues 3:** Formatting toolbar->after selecting rows and columns to create table and inserting table in edit field->table properties popup not clearly visible.

**Issue 4:** Formatting toolbar->before entering anything in edit field->table icon pops up->table icon not clearly visible.

**What is the expected behavior?**
Icons should be clearly visible in High Contrast 'Aquatic' mode.

**Which versions of TinyMCE, and which browser / OS are affected by this issue? Did this work in previous versions of TinyMCE?**
OS Version: Windows 11 Dev 22H2(OS Build 22621.1265 )
Edge Dev Browser Version: 110.0.1587.57 (Official build) (64-bit)
Tiny MCE version V6.0
![toolbar_menu popup_expand icon_normal mode](https://user-images.githubusercontent.com/93735775/222413745-71d4e395-68b6-46c3-89e7-f07c35fc3027.png)
![toolbar_menu popup_expand icon_aquatic mode](https://user-images.githubusercontent.com/93735775/222413752-b47fa551-e05d-4ef2-a059-b6cb9730ac25.png)
![table properties_dropdown icon_normal mode](https://user-images.githubusercontent.com/93735775/222413759-022e47ea-6f72-4aec-8210-cc7f5fb5d646.png)
![table properties_dropdown icon_aquatic mode](https://user-images.githubusercontent.com/93735775/222413762-b78bfe72-5eef-4f31-b83c-20bcc24263ca.png)
![table option icons_aquatic mode](https://user-images.githubusercontent.com/93735775/222413764-4fd93eb1-6341-47a4-919c-85cb279157fc.png)
![table option icons_normal mode](https://user-images.githubusercontent.com/93735775/222413766-f3edc906-f85d-4c72-a6bb-9170a5532f26.png)
![table icon_aquatic mode](https://user-images.githubusercontent.com/93735775/222415175-a23b81d3-94fd-4511-b919-f63d61e0cf1f.png)
![table icon_normal mode](https://user-images.githubusercontent.com/93735775/222415180-746568ca-3d6f-464b-a43e-d1aac43881d5.png)

**Comment by msftedad** [comment]
Hi team, Any update on this issue?

**Comment by msftedad** [comment]
Hi Team,

Please respond to the comments.

Thanks

**Comment by tiny-ci-servers** [comment]
This issue is stale because it has been open 365 days with no activity. Please comment if you wish to keep this issue open or it will be closed in 30 days.
